### DISCUSSION TITLE
showing meeting cities on list

### DISCUSSION THREAD

**Discussion by CheriJones** [code4recovery/12-step-meeting-list#1439] [Q&A] [answered]
### Contact Details

theempressjones@cyberservices.com

### Website With Issue

wamo-aa.org

### What happened?

when code is added to function.php to show cities in the meeting list there isn't a space between the address and the city - 
![image](https://github.com/code4recovery/12-step-meeting-list/assets/88801930/d47b6674-b7bb-4be6-932a-954688517e11)

### What browsers are you seeing the problem on?

Firefox, Chrome

**Comment by kiyote33** [comment]
Hi @CheriJones, what's your function code look like?

**Reply by CheriJones** [reply]
$tsml_street_only = false;

**Reply by CheriJones** [reply]
added to functions.php

**Reply by kiyote33** [reply]
Yea. No help here. Our default behavior looks like this:
![image](https://github.com/code4recovery/12-step-meeting-list/assets/57125427/12c271ae-2e7e-47d5-8fac-4327236166d6)

**Comment by joshreisner** [comment]
hi @CheriJones - looks like your site has some custom CSS that hides `<br>` tags. when i comment that out then the line breaks show up in the list

i'd try taking that out. looks like it was added using the `wp-custom-css` plugin

<img width="1280" alt="Screenshot 2024-06-02 at 8 39 45 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/ca00d17f-770d-46be-9241-801c4619e448">

**Reply by CheriJones** [reply]
@joshreisner total novice here - can't find a plugin wp-custotm-css - can you tell me a file to go to or a little more detail how to fix this - sorry for all the bother

**Reply by joshreisner** [reply]
i think it would be in the wp dashboard, rather than a file. i would look for a "Custom CSS" option in the sidebar?

**Reply by CheriJones** [reply]
i added to themes function.php should i have added to 12-step-meeting-list/includes/functions.php ?? - from image above it looks like i should comment out that line in meetings file but i can't seem to find that file

**Reply by CheriJones** [reply]
nothing in sidebar - but the company that designed this has burried a lot of stuff - here is image of side bar - suggestions ??
![image](https://github.com/code4recovery/12-step-meeting-list/assets/88801930/697d9fc5-834a-41d3-b168-1fc58ca9d2ad)

**Reply by SIA-WEBSUP** [reply]
I would check under the tools... some plugins simply add something to that 'applet' in the sidebar.

**Reply by SIA-WEBSUP** [reply, marked as answer]
You might also check Appearance->Customize->Additional CSS

**Reply by CheriJones** [reply]
got it under additional CSS - thank you all so much for your help
